docbook/reference/en/en-US/modules/RESTEasy_Client_Framework.xml

```diff
@@ -234,67 +234,188 @@ public interface LibraryService {
    
    <sect1 id="transport_layer">
      <title>Apache HTTP Client 4.x and other backends</title>
-     
+
+       <para>
+           Network communication between the client and server is handled by default in Resteasy.
+           It uses the
+           <code class="classname">HttpClient</code> from the Apache
+           <code class="classname">HttpComponents</code> project.
+           The interface between the Resteasy Client Framework and the network
+           is defined by Resteasy's

```

Comment:
I have changed this in the file only.  However I was only following the existing style.  All references in this file to the project was presented as "Resteasy"  In addition in many other chapters the text is, "Resteasy" and not "RESTEasy".